android/quest/build.gradle.kts

```diff
@@ -361,8 +370,79 @@ dependencies {
   androidTestImplementation(libs.ui.test.junit4)
   androidTestImplementation(libs.hilt.android.testing)
   androidTestImplementation(libs.mockk.android)
+  androidTestImplementation("androidx.benchmark:benchmark-junit4:1.1.1")
   ktlint(libs.ktlint.main) {
     attributes { attribute(Bundling.BUNDLING_ATTRIBUTE, objects.named(Bundling.EXTERNAL)) }
   }
   ktlint(project(":linting"))
 }
+
+task("evaluatePerformanceBenchmarkResults") {
```

Comment:
Add documentation for this

Reply:
inline docs are done, I'm going to add docs on perf tests overall...

Reply:
Thanks for this